web/styles/settings.css

```diff
@@ -1504,6 +1505,26 @@ $option_title_width: 180px;
     }
 }
 
+#realm_jitsi_server_url_setting {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 15px;
+
+    select {
+        min-width: auto;
+        text-overflow: ellipsis;
+    }
+
+    & input,
+    select {
+        width: 311px;
```

Comment:
Where does this 311 come from? A `calc()` or comment or CSS variable would usually be better.

Reply:
I did some refactoring with the style, using the existing class `setting_url_input`, so we no longer need to define width here. I also made some design changes; I think labels on a separate line look better. Below are the before and after screenshots.

**Before:**

![](https://user-images.githubusercontent.com/87542880/261873935-bf95392d-dfc7-4186-b79b-ad5fdcc93bd3.png)

**After:**

![Screenshot 2023-09-20 001026](https://github.com/zulip/zulip/assets/87542880/a5f591e3-3167-49ed-a42c-0e90082bcf47)

![Screenshot 2023-09-20 001051](https://github.com/zulip/zulip/assets/87542880/420fcd75-4d37-4a79-bce7-ee43dbc49351)